p5/core/primitives.py

```diff
@@ -558,23 +583,32 @@ def arc(*args, mode=None, ellipse_mode=None):
         raise ValueError("Unexpected number of arguments passed to arc()")
 
     if ellipse_mode is None:
-        emode = _ellipse_mode
+        emode = p5.renderer.style.ellipse_mode
     else:
         emode = ellipse_mode
 
-    if emode == 'CORNER':
-        corner = Point(*coordinate)
-        dim = Point(width / 2, height / 2)
-        center = (corner.x + dim.x, corner.y + dim.y, corner.z)
-    elif emode == 'CENTER':
-        center = Point(*coordinate)
-        dim = Point(width / 2, height / 2)
-    elif emode == 'RADIUS':
-        center = Point(*coordinate)
-        dim = Point(width, height)
-    else:
-        raise ValueError("Unknown arc mode {}".format(emode))
-    p5.renderer.arc(center, dim, start_angle, stop_angle, mode)
+    if builtins.current_renderer == 'vispy':
+        if emode == 'CORNER':
+            corner = Point(*coordinate)
+            dim = Point(width / 2, height / 2)
+            center = (corner.x + dim.x, corner.y + dim.y, corner.z)
+        elif emode == 'CENTER':
+            center = Point(*coordinate)
+            dim = Point(width / 2, height / 2)
+        elif emode == 'RADIUS':
+            center = Point(*coordinate)
+            dim = Point(width, height)
+        else:
+            raise ValueError("Unknown arc mode {}".format(emode))
+        p5.renderer.arc(center, dim, start_angle, stop_angle, mode)
+
+    elif builtins.current_renderer == 'skia':
+        if not should_draw() or start_angle == stop_angle:
+            return
+        x, y = coordinate[0], coordinate[1]
+        vals = mode_adjust(x, y, width, height, emode)
```

Comment:
It seems that the documentation for `arc` only has `{'CORNER', 'CENTER', 'RADIUS'}`, but `module_adjust` also supports `CORNERS` as a potential input. Should we sanitize the input here, or is the documentation missing a mode?

Reply:
The documentation is missing a mode. 
[ellipseMode](https://p5js.org/reference/#/p5/ellipseMode
)
[rectMode](https://p5js.org/reference/#/p5/rectMode
)